.github/pr_labeler.yml

```diff
@@ -1,3 +1,9 @@
+cli:
```

Comment:
I'm not sure this change is needed. If we do this it creates the impression that you need to add the entry sorted/at the right place. Also doesn't seem related to this PR

Reply:
will revert then